core-bundle/src/EventListener/DataContainer/JobsListener.php

```diff
@@ -65,23 +65,34 @@ public function onAttachmentsCallback(DataContainerOperation $operation): void
         }
 
         $attachments = $this->jobs->getAttachments($job);
-        $numberOfAttachments = \count($attachments);
 
-        if (0 === $numberOfAttachments) {
+        if ([] === $attachments) {
             $operation->hide();
 
             return;
         }
 
-        // TODO: we need a template and Stimulus logic to have an operation with sub
-        // operations just like the [...] in the current context menu to be able to
-        // display more than just one download
-        $attachment = $attachments[0];
-
-        $operation['icon'] = 'theme_import.svg';
-        $operation['title'] = $attachment->getFileLabel();
+        $operations = [];
+
+        foreach ($attachments as $attachment) {
+            $operations[] = new DataContainerOperation(
+                'download',
+                [
+                    'icon' => 'theme_import.svg',
```

Comment:
can we get the file type icon here instead?

Reply:
Why the file type icon? A download icon from Lucide is more suitable imho (which this already is).

Reply:
Agreed. As I said, the individual downloads probably don't need an icon at all.

Reply:
We could also pack the downloads into a ZIP archive and not provide individual downloads at all. WDYT?

Reply:
I would not name them after any Contao specific action anyway. Personally, I would keep them just the same as they are named in Lucide. So that would be just `download`. Whether it's used in the theme export/import, jobs, elsewhere doesn't matter. 
But I don't want to rename icons in this PR - imho this should happen separately. Changing to `theme_import.svg` again now.

Reply:
> I would not name them after any Contao specific action anyway.

Agreed - but they need to for backwards compatibility.